src/egon/data/datasets/ch4_prod.py

```diff
@@ -198,7 +198,8 @@ def load_biogas_generators(scn_name):
     )
     target_file = Path(".") / "datasets" / "gas_data" / basename
 
-    urlretrieve(url, target_file)
+    if not target_file.is_file():

```

Comment:
In our current setting, that would mean we should manually copy-paste the file before this task is executed (since the file is not online anymore), right?

Reply:
I think they replaced the file with a newer version (see https://github.com/openego/powerd-data/issues/337#issuecomment-2498004910)